lucille-plugins/lucille-tika/src/test/java/com/kmwllc/lucille/tika/stage/TextExtractorTest.java

```diff
@@ -90,7 +110,13 @@ public void testExcel() throws StageException {
   public void testCustomTikaConfig() throws StageException {
     Stage stage = factory.get("TextExtractorTest/tika-config.conf");
     Document doc = Document.create("doc1");
-    doc.setField("path", "src/test/resources/TextExtractorTest/tika.txt");
+
+    // get current directory absolute path
+    Path currentRelativePath = Paths.get("");
```

Comment:
try creating a Path from the relative path string ("src/test/...") and then calling toAbsolutePath() on it -- can do this in one line

Reply:
resolved in new commit